packages/connector-types/package.json

```diff
@@ -0,0 +1,54 @@
+{
+  "name": "@logto/connector-types",
+  "version": "0.1.0",
+  "main": "lib/index.js",
+  "author": "Logto Team",
+  "license": "MPL-2.0",
+  "files": [
+    "lib"
+  ],
+  "private": true,
+  "scripts": {
+    "preinstall": "npx only-allow pnpm",
+    "precommit": "lint-staged",
+    "build": "rm -rf lib/ && tsc --p tsconfig.build.json",
```

Comment:
`lib/` should be `build/`

See `packages/connector-*/tsconfig.base.json`

```json
{
  "extends": "@silverhand/ts-config/tsconfig.base",
  "compilerOptions": {
    "outDir": "build",
  ……
```

Reply:
this should be `lib/` actually?